modules/installation-configuration-parameters.adoc

```diff
@@ -1,54 +1,110 @@
 
 |`provider`
 |The cloud provider to host the cluster.
-a|Enter one of the following values:
+|`aws`
+
+|`baseDomain`
+|The base domain of your cloud provider. This value is used to create routes
+to your {product-title} cluster components.
+|A fully-qualified domain or subdomain name, such as `dev.example.com`.
+
+|`name`
+|The name of your cluster.
+|A string that contains uppercase or lowercase letters, such as `dev`.
+
+|`region`
+|The name of your cluster.
+|A valid AWS region, such as `us-east-1`.
 
-* aws
-* azure
-* openstack
+|`pullSecret`
+|The pull secret that you obtained from link:try.openshift.com[the OpenShift start page].
+|
+[source,json]
+----
+{  
+   "auths":{  
+      "cloud.openshift.com":{  
+         "auth":"b3Blb=",
+         "email":"you@example.com"
+      },
+      "quay.io":{  
+         "auth":"b3Blb=",
+         "email":"you@example.com"
+      }
+   }
+}
+----
 
-|`base-domain`
-|The base domain of your cloud provider. This value is used to <do a thing>.
-|A FQDN, such as `dev.example.com`.
+|`sshKey`
+|The SSH key to use to access your cluster machines.
+|A valid public SSH key.
 |====
 
-.All installation parameters
-[cols=".^3,.^5,.^5a,.^5",options="header"]
+
+.Optional AWS platform parameters
+[cols=".^2,.^3,.^3a",options="header"]
 |====
-|Parameter|Description|Value format|Default value
+|Parameter|Description|Values
 
-|`provider`
-|The cloud provider to host the cluster.
-a|Enter one of the following values:
+|`machines.platform.aws.iamRoleName`
+|The IAM role assigned to the machines in the pool.
+|
+
+|`machines.platform.aws.rootVolume.iops`
+|The Input/Output Operations Per Second (IOPS) that is reserved for the root volume.
+|Integer, for example `4000`
+
+|`machines.platform.aws.rootVolume.size`
+|The size in GiB of the root volume.
+|Integer, for example `500`
+
+|`machines.platform.aws.rootVolume.type`
+|The instance type of the root volume.
+|Valid link:https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/EBSVolumeTypes.html[AWS EBS instance type],
+such as `io1`.
+
+|`machines.platform.aws.type`
+|The EC2 instance type for the control plane machines.
+|Valid link:https://aws.amazon.com/ec2/instance-types/c5/[AWS instance type],
+such as `c5.9xlarge`.
+
+|`machines.platform.aws.zones`
+|The availability zones where the installation program creates machines for this
+pool.
+|Valid AWS availability zone, such as `us-east-1c`.
 
-* aws
-* azure
-* openstack
-| n/a
+|`platform.aws.region`
+|The AWS region that the installation program creates resources in.
+|Valid link:https://docs.aws.amazon.com/general/latest/gr/rande.html[AWS region],
+such as `us-east-1`
 
-|`base-domain`
-|The base domain of your cloud provider. This value is used to <do a thing>.
-|A FQDN, such as `dev.example.com`.
-|n/a
+|`platform.aws.userTags`
+|A map of keys and values that the installation program adds as tags to all
+resources that it creates.
+|Key value pairs in the `<key>: <value>` format. For more information about
```

Comment:
They don't have to have this syntax, they just have to be [YAML mappings][1].  The format you describe is a [block mapping][2], but you could use [flow mappings][3] too.  Probably best to just drop the format sentence and leave it to your earlier "A map..." and the example YAML to get the point across, without wading into YAML syntax.

[1]: https://yaml.org/spec/1.2/spec.html#mapping//
[2]: https://yaml.org/spec/1.2/spec.html#style/block/mapping
[3]: https://yaml.org/spec/1.2/spec.html#id2790832

Reply:
Because it's often helpful to provide an example, and in this draft, the sample YAML isn't visible on the same screen.

Reply:
Your new "Any valid YAML map, such as..." wording looks fine to me, although I don't seem to be authorized to mark this thread resolved.